packages/orchestrator-ui-components/src/hooks/useDataDisplayParams.ts

```diff
@@ -12,7 +12,7 @@ import { DEFAULT_PAGE_SIZE } from '../components';
 export type DataDisplayParams<Type> = {
     pageSize: number;
     pageIndex: number;
-    sortBy?: GraphQLSort<Type>;
+    sortBy: GraphQLSort<Type>;
```

Comment:
I made set this to a possible undefined value since the data can be entered trough the query paramter and can potentially be missing or invalid. In case of missing or invalid it is undefined in the app. Or do you have some fallback mechanism while processing the parameters?

Reply:
I give a default sorting with every page. That way it will always get a value even if it's not provided. Is that what you mean or is it something else? If it's optional I get errors in other places although I don't remember exactly where